webpack.config.dev.js

```diff
@@ -50,6 +60,5 @@ export default {
       {test: /(\.css|\.scss)$/, loaders: ['style', 'css?sourceMap', 'postcss', 'sass?sourceMap']},
```

Comment:
There is a way to support emitting it, but you need to use `resolveLoader.moduleExtensions`. See https://webpack.js.org/guides/migrating/#automatic-loader-module-name-extension-removed